jina/jaml/parsers/default/v1.py

```diff
@@ -13,6 +13,7 @@ def parse(self, cls: Type['JAMLCompatible'], data: Dict) -> 'JAMLCompatible':
         :param cls: target class type to parse into, must be a :class:`JAMLCompatible` type
         :param data: flow yaml file loaded as python dict
         """
+        data = JAML.expand_dict(data, None)
```

Comment:
can we use a different variable to avoid updating variables in place?

Reply:
done